packages/ds-healthcare-gov/src/components/Footer/__snapshots__/InlineLinkLists.test.tsx.snap

```diff
@@ -76,7 +76,7 @@ exports[`InlineLinkLists renders lists of links 1`] = `
         class="hc-c-footer__inline-item ds-u-margin-y--0 ds-u-display--inline-block"
       >
         <button>
-          footer.privacySettings
+          Privacy Settings
```

Comment:
Note to self: We may want to go in and remove `t` props in favor of just calling our translation hook inside these components. It would make the tests simpler and be less data to pass around. All the other labels in this snapshot are produced by a dumb mock translation function.